### PR TITLE
login screen front end issue

### LINKED ISSUE #196: Login Screen
The solution to this issue is expected to:
* implement Login design according to Figma
* implement gql to hit the backend API to verify the user logging in, to authenticate and:
  * In case the user is not present, show the corresponding error message.
  * In case the password doesn't match, show the corresponding error message.
  * In case the user is not verified, show the `User not verified` page.
  * In case the user passes the above checks, direct the user further

Use this [link](https://www.figma.com/file/IuUnvGJc0ZqjUsnXF0DtFon3/MentorFix?node-id=21%3A10)
Implement screen 1.3 or 1.2

### PR DESCRIPTION
**Pull request by NikitaMasand**
fixes #196 

Following are the ss of the login page:
mf1
mf2

### CODE REVIEW

**Comment by NikitaMasand** [comment]
@abhishek71994  please review

File: src-view/assets/styles/LoginView.module.scss [hunk 1]
```diff
@@ -1,49 +1,87 @@
...
-}
+@import './mixins';
+
+.loginpage {
+
+    .bottomright {
+        >img {
+            display: flex;
+            float: right;
+            width: 300px;
```
Comment (abhishek71994) [inline src-view/assets/styles/LoginView.module.scss]: Change the `px` to `rem` please

**Review by abhishek71994 — APPROVED** [review]
Looks good to me. Just change the `px` to `rem`. As `rem` depends on the relative size of 16px. Pixel can be different for different screens to `rem` helps keep the styling consistent

File: src-view/assets/styles/LoginView.module.scss (hunk 1 above)
Comment (NikitaMasand) [reply src-view/assets/styles/LoginView.module.scss]: got it. changed it

**Comment by NikitaMasand** [comment]
@abhishek71994 please check. I have changed the dimensions to rem as suggested.

File: src-view/assets/styles/LoginView.module.scss [hunk 2]
```diff
@@ -1,49 +1,87 @@
...
+            padding-left: 200px;
+            width: 78%;
+            height: 60%;
+        }
+    
+        .forgotcontent {
+            color: #555555;
+            padding-left: 30%;
+            padding-top: 10px;
+            font-size: 14px;
```
Comment (abhishek71994) [inline src-view/assets/styles/LoginView.module.scss]: In all the files I mean.

**Review by abhishek71994 — CHANGES_REQUESTED** [review]

File: src-view/assets/styles/LoginView.module.scss (hunk 2 above)
Comment (abhishek71994) [reply src-view/assets/styles/LoginView.module.scss]: Like all px's except maybe `border-radius`

**Comment by NikitaMasand** [comment]
updated ss after changes:
mf1
mf2

**Review by abhishek71994 — APPROVED** [review]

**Pull request merged**